controller/reconciler.go

```diff
@@ -42,11 +42,11 @@ func Reconcile(ctx reconciler.Context, client *versionedclient.Clientset, patch
 		}
 	} else if oputil.Contains(patch.Finalizers, finalizerName) {
 		if err := updateTarget(client, patch, true); err != nil {
-			if kerr.IsNotFound(err) {
-				// do not need to panic just log output
-				ctx.Logger().Info("Virtual service not found.")
+			if !kerr.IsNotFound(err) {
```

Comment:
seems like you can flip the statements and not need the `!` here? double negatives are harder to follow